src/jestRunnerConfig.ts

```diff
@@ -41,7 +41,18 @@ export class JestRunnerConfig {
   }
 
   public get projectPath(): string {
-    return vscode.workspace.getConfiguration().get('jestrunner.projectPath') || this.currentWorkspaceFolderPath;
+    let projPath: string = vscode.workspace.getConfiguration().get('jestrunner.projectPath');
+    if (projPath) {
+      return projPath;
+    }
+
+    const configPath = this.findConfigPath();
+    const configDir = path.dirname(configPath);
+    if (fs.existsSync(path.join(configDir, 'node_modules'))) {
```

Comment:
check for the jest binrary. in yarn workspaces each package has a node_modules folder with symlinks but the jest binrary is maybe only available in the root package.json

Reply:
Would that not be better done in `jestBinPath`? That function currently reuses `projectPath` but another commit/PR could address that problem.

`node_modules` may be the wrong thing for to check for here, but I don't think the jest binary is the right thing. The problem that prompted this PR was missing `babel.config.js`, so maybe we should check for that here? What else might `jest` look for?